scripts/xbstrap

```diff
@@ -1542,6 +1609,9 @@ class Plan:
 			item.build_edges.add((action.BUILD_PKG, subject))
 
 		elif action == Action.RUN:
+			add_tool_dependencies(subject)
```

Comment:
We might also want `add_implicit_pkgs()` and `add_pkg_dependencies(subject)` (just not the build_edges).